drivers/include/disp_dev.h

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (C) 2020 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    sys_disp_dev Display device generic API
+ * @ingroup     sys
+ * @brief       Define the generic API of a display device
+ * @{
+ *
```

Comment:
Should we add `@experimental`, or some note that the API is new and might change?

Reply:
I have no opinion here. Once this gets merged, #13124 will be the first user so not really experimental anymore.

Reply:
IMO we should give all new API's a six month (or more) period for maturing. Reduces the pain for users, if they stick to "non-new" APIs.

Whether this is called @experimental, or announced as "new, might change" or whatever, I don't really care. Just something for managing expectations, and not locking us into an API we might want to change.

lvgl will be an "internal user", so it wouldn't count when considering an API change.

Reply:
Added the `@experimental` keyword.